pdc_client/__init__.py

```diff
@@ -125,7 +125,10 @@ class to perform requests.
             if insecure is not None:
                 sys.stderr.write("Warning: '%s' option is deprecated; please use '%s' instead\n" % (
                     CONFIG_INSECURE_KEY_NAME, CONFIG_SSL_VERIFY_KEY_NAME))
-                ssl_verify = not insecure
+                if insecure:
+                    ssl_verify = not insecure
+                else:
+                    ssl_verify = ssl_verify or not insecure
```

Comment:
From line 131 if ssl_verify set as fase, it will not take effect and use the value `not insecure`, that should be true here. So ssl_verify will be true in line 131 no matter what ssl_verify is set in cofig. Is it what you expect? If it is, seems a little complicate to do it. Could simple `ssl_verify=true` do the same thing?